dist/index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Morning To DO List</title>
+  <script defer src="main.js"></script></head>
```

Comment:
I will like to know, why did you use `defer` here?

Reply:
I am not supposed to review the dist folder, sorry for that. Apart from this comment, all other comments should be applied to the index.html file from the src folder